### PR TITLE
Hybrid Analysis

### PR DESCRIPTION
**Pull request by dantavori**
## Status
Ready

## Related Issues
fixes: [https://github.com/demisto/etc/issues/18693](https://github.com/demisto/etc/issues/18693)

## Description
  - Added **jobID**, **sha256** and **environmentID** arguments to the ***hybrid-analysis-get-report-status*** command.
  - Added the **malicious_threat_levels** argument to the ***hybrid-analysis-detonate-file*** command.
  - The ***hybrid-analysis-detonate-file*** command should now run as expected.

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests
- [ ] Documentation (with link to it)
- [x] Code Review

## Technical writer review
Mention and link to the files that require a technical writer review.
- [ ] [YAML file](https://github.com/demisto/content/blob/HybridAnalysisBug/Integrations/integration-HybridAnalysis.yml)
- [ ] [CHANGELOG](https://github.com/demisto/content/blob/HybridAnalysisBug/Integrations/integration-HybridAnalysis_CHANGELOG.md)

### CODE REVIEW

**Comment by ronykoz** [comment]
@anara123 FYI breaking BC here, changing an arg to be required(if you don't provide it, it will go to a dead end code which lead to this problem to begin with)

**Comment by anara123** [comment]
@ronykoz so I understand, the command cant run without environmentid anyway. If that is the case, sure will merge as soon as @kirbles19 docs-approves this pr

File: Integrations/integration-HybridAnalysis.yml [hunk 1]
```diff
@@ -942,9 +926,84 @@ script:
...
     description: Submits a file for detonation in Hybrid Analysis.
   - name: hybrid-analysis-get-report-status
-    arguments: []
+    arguments:
+    - name: jobID
+      description: The Job ID of the submission
+    - name: sha256
+      description: The submission SHA256
+    - name: environmentID
+      description: The environmentID of the submission
```
Comment (kirbles19) [inline Integrations/integration-HybridAnalysis.yml]: Add periods to the descriptions.

File: Integrations/integration-HybridAnalysis.yml [hunk 2]
```diff
@@ -942,9 +926,84 @@ script:
...
+      type: string
+    - contextPath: DBotScore.Score
+      description: The DBot score.
+      type: number
     description: Submits a file for detonation in Hybrid Analysis.
   - name: hybrid-analysis-get-report-status
-    arguments: []
+    arguments:
+    - name: jobID
+      description: The Job ID of the submission
```
Comment (dantavori) [inline Integrations/integration-HybridAnalysis.yml]:
```suggestion
      description: The Job ID of the submission.
```

File: Integrations/integration-HybridAnalysis.yml [hunk 3]
```diff
@@ -942,9 +926,84 @@ script:
...
+    - name: sha256
+      description: The submission SHA256
```
Comment (dantavori) [inline Integrations/integration-HybridAnalysis.yml]:
```suggestion
      description: The submission SHA256.
```

**Comment by dantavori** [comment]
@anara123 can we merge then?

**Comment by anara123** [comment]
yes

**Pull request merged**